ple/games/donkeykong/Board.py

```diff
@@ -0,0 +1,359 @@
+__author__ = 'Batchu Vishal'
+import pygame
+import math
+import random
+import sys
+
+from pygame import mixer
+from Person import Person
+from OnBoard import OnBoard
+from Coin import Coin
+from Player import Player
+from Fireball import Fireball
+from DonkeyKong import DonkeyKong
+from Button import Button
+
+# Initialize the mixer
+mixer.init()
```

Comment:
Can you remove all sounds in the game? Along with their files? Sound isnt used by PLE